src/lua/luaLoggerBinding.c

```diff
@@ -977,6 +979,257 @@ static int lua_update_gps(lua_State *L)
         return 0;
 }
 
+static int lua_sx_update_linear_graph(lua_State *L)
+{
+        lua_validate_args_count(L, 0, 1);
+        if (lua_gettop(L) > 0) {
+                lua_validate_arg_number(L, 1);
+                uint16_t value = lua_tointeger(L, 1);
```

Comment:
Any check for overflow here?

Reply:
Probably not a issue due to implicit casting.

Reply:
Yeah, it won't be crashy if they send bad data, ShiftX3 just won't work as expected.